tests/http_bench.ts

```diff
@@ -0,0 +1,22 @@
+// Used for benchmarking Deno's networking. See tools/http_benchmark.py
+// TODO Replace this with a real HTTP server once
+// https://github.com/denoland/deno/issues/726 is completed.
+import * as deno from "deno";
+let addr = deno.args[1] || "127.0.0.1:4500";
+let listener = deno.listen("tcp", addr);
+const response = new TextEncoder().encode(
+  "HTTP/1.1 200 OK\r\nContent-Length: 12\r\n\r\nHello World\n"
+);
+
+async function main() {
+  console.log("Listening on", addr);
+  while (true) {
+    const conn = await listener.accept();
```

Comment:
This does only 1 connection in parallel. That's not very realistic.

Reply:
No. You await the read and write op before accept() gets called again.

simple fix:

```
while(true) {
  handleConnection(await listener.accept())
}

async function handleConnection(conn) {
  await conn.read(...)
  await conn.write(...)
  etc
}

```

Reply:
oops - was on my phone for the last reply. fixed.